### ISSUE TITLE
Site Not Found

### ISSUE THREAD

**Issue by alptemiz** [yuryzablotski/yuzaR-Blog#3]
Hey mate,
Unfortunately your lovely blog is down. 
Just wanted to let you know, I hope it will be back soon.

**Comment by yuryzablotski** [comment]
Hey mate, I know, but thanks for info. The service provide want me to pay a lot since my truffic increased. And since I don't earn anithing from my blog and do actually blog about open source, I can't pay their price. So, I need to figure out the solution for it, but until then the youtube is online and will stay this way I hope. The blog is actually the script for youtube, so that the videos are almost identical to the blog. I hope that helps. You can also reach me better in youtube comments than here. Cheers!

**Comment by yuliaUU** [comment]
just as suggestion: may be smth like https://jupyterbook.org/en/stable/intro.html will be usefull, since you already got everything as Rmd files, that would be easy to convert to jupyterlab book and it just hosted on teh GitHub.io website: so it is free

I am also willing to help converting to this to the jupyterlab if needed- since I used your blog a lot to learn all the topics and they were extreemly helpfull for me:)

**Comment by yuryzablotski** [comment]
hosting on GitHub.io website (when possible without Netlify or similar) is actually a great idea, thanks! I just wander whether it's possible for something R like? Because I checked the link you send and have to say that I don't use command line or python/jupiter (don't have much time to learn and go through trial and errors, so at the moment I am horrified by them :). I'll check how other R developers host their books online for free. Or do you know how it can be done in R? 

By the way, thanks for the feedback! I am glad my content is useful and would love to create more with time.

**Comment by yuliaUU** [comment]
Yes smth like Jekyll seems to work https://chadbaldwin.net/2021/03/14/how-to-build-a-sql-blog.html but I never tried it

**Comment by yuliaUU** [comment]
also Notion just announces Notion websites ( Notion is free) https://www.notion.so/help/guides/build-a-website-with-notion-in-seconds-no-coding-required

more info here: https://www.notion.so/help/public-pages-and-web-publishing
